### DISCUSSION TITLE
Review/cleanup of "Who uses mithril"

### DISCUSSION THREAD

**Discussion by webketje** [MithrilJS/mithril.js#2953] [Meta/Feedback]
It seems like a lot of the examples in _who uses mithril_ are outdated, discontinued, or have changed technologies.
I wanted to be able to see some client code in the network requests/ on-page but failed... So here's a list of the URL's I checked (checks done by opening multiple pages, observing routing, searching for 'mithril' in JS request bundles) :

No longer working or replaced:

- [Autodesk Stingray Engine](https://www.autodesk.com/products/stingray/overview): redirected to a notice that the software is discontinued by Autodesk.
- [SkunkTime](http://www.skunktime.com/): Domain no longer active
- [ListKit](http://www.listkit.co/): Server not found/ Github inexistant
- [EquityEats](https://www.equityeats.com/): Server sends back 'Not Found'
- [Nike Github](http://nike-inc.github.io/): redirects to https://engineering.nike.com/, made with Gatsby (React)
- [MLB Draft Tracker 2016](http://m.mlb.com/draft/tracker/): App displays no results, unable to find mithril traces in network requests or on-page. Seems to have been replaced with React in footer/header.
- [90min.com](https://www.90min.com/): Seems to have been replaced with React (in vendor bundle, no traces of mithril).
- [Apcera](http://www.apcera.com/) Redirects to https://www.ericsson.com/en/digital-services/edge-computing
- [Fusion TV](https://fusion.tv/show/the-chris-gethard-show/): 404 Not Found
- [Abyss](https://abyss.online/): Server Not Found
- [dstllry.co](http://www.dstllry.co/): Server not found/ Github inexistant
- [Curate Science](http://www.curatescience.org.s3-website-us-east-1.amazonaws.com/#/login): 404 Not Found, root URL uses React
- [RacingExtinction.com](http://racingextinction.com/): looks like a simple Wordpress, no traces of mithril.

Verified using mithril:

- [Calligraphr](https://www.calligraphr.com/) (only  visible after registration)
- [Sunday Seven](https://www.sunday-seven.com/)
- [Minetest mapserver](http://github.com/thomasrudin-mt/mapserver/): under `/static` folder
- [Flarum.org](http://flarum.org/)
- [Melpa.org](https://melpa.org)
- [LikeMyShop](https://www.likemy.shop/) (testable at https://likemy.shop/example, might wanna add that)
- [Maildeveloper](https://maildeveloper.com/demo)
- [RayFeed](https://rayfeed.com/)

If I get more time I'll check the rest

**Comment by cceconi** [comment]
You can add the INPI website (French National Institute which register and manage your trademark and patent). Here the entry point link : [https://procedures.inpi.fr/?/](https://procedures.inpi.fr/?/). Under, there is more than 8 MithrilJS based webapps. 
I'm the one who choose MithrilJS because it rules 😁. Keep the goob job

**Comment by dead-claudia** [comment]
That wiki page *should* be editable by any registered GitHub user, so feel free to edit that. If you can't edit the wiki, let me know here, and I can rectify that.

**Comment by jcs224** [comment]
@isiahmeadows @webketje I can edit it, which probably means anyone can 😄

**Comment by webketje** [comment]
@jcs224 @isiahmeadows My idea was to finish the review and edit it at once.
I will remove the ones with HTTP error codes, move the more _palpable_ projects (those in which you can clearly notice Mithril) to the top, and add some info about how to see it in action where applicable. 

Perhaps for the ones that used to use it, I could put a title at the bottom "Used previously at.." but not sure what do you think?

**Comment by jcs224** [comment]
@webketje I don't know if "used previously at" is a great thing to advertise, seems desperate IMO. We should try soliciting more input from people who are happily using it today.

**Comment by dead-claudia** [comment]
@webketje 👍 Good with me. I'll let this issue sit for discussion, then.

**Comment by cmawhorter** [comment]
i'd recommend some guidance on how to add new entries.  i have one to add but putting it at the top feels wrong so i'll hold off.  

fwiw if i don't make it back this way here's a [custom tshirt designer](https://www.designashirt.com/design-studio) i built in a little over a month of solid work.  it's still mithril 1.x but has been in service for years and servicing lots of users.

**Comment by dead-claudia** [comment]
@cmawhorter Feel free to put it wherever it makes you feel comfortable, following existing patterns within the page.
